Global.gd

```diff
@@ -67,7 +67,7 @@ func goto_scene(target_scene):
 	# we can be sure that no code from the current scene is running:
 	match target_scene:
 		Scene.OVERWORLD:
-			call_deferred("_deferred_goto_scene", Scene.OVERWORLD, "res://Overworld.tscn")
+			call_deferred("_deferred_goto_scene", Scene.OVERWORLD, "res://overworld/Overworld.tscn")
```

Comment:
Glad to see overworld moved to its own directory.

Reply:
fixed